packages/cadl-ranch-specs/http/parameters/spread/main.tsp

```diff
@@ -37,6 +37,126 @@ namespace Model {
   @route("/request-body")
   @put
   op spreadAsRequestBody(...BodyParameter): NoContentResponse;
+
+  @doc("This is a model only with `@body` property.")
+  model CompositeRequestOnlyWithBody {
+    @body body: BodyParameter;
+  }
+
+  @scenario
+  @scenarioDoc("""
+  Test case for spread model only with `@body` property. 
+  
+  Should generate request body model named `BodyParameter`.
+  Should not generate model named `CompositeRequestOnlyWithBody`.
+  Should generate an operation like below:
+  ```
+  spreadCompositeRequestOnlyWithBody(bodyParameter: BodyParameter)
+  ```
+  Note the parameter name is guessed from the model name and it may vary by language.
+  
+  Expected request body:
+  ```json
+  { "name": "foo" }
+  ```
+  """)
+  @route("/composite-request-only-with-body")
+  @put
+  op spreadCompositeRequestOnlyWithBody(...CompositeRequestOnlyWithBody): NoContentResponse;
+
+  @doc("This is a model without `@body` property.")
+  model CompositeRequestWithoutBody {
+    @path
+    name: string;
+
+    @header
+    testHeader: string;
+  }
+
+  @scenario
+  @scenarioDoc("""
+  Test case for spread model without `@body` property. 
+  
+  Should not generate model named `CompositeRequestOnlyWithBody`.
+  Should generate an operation like below:
+  ```
+  spreadCompositeRequestWithoutBody(name: string, testHeader: string)
+  ```
+
+  Expected path parameter: name="foo"
+  Expected header parameter: testHeader="bar"
+  """)
+  @route("/composite-request-without-body/{name}")
+  @put
+  op spreadCompositeRequestWithoutBody(...CompositeRequestWithoutBody): NoContentResponse;
```

Comment:
I don't think this behavior has anything to do with spread.

I assume this is existing emitter behavior of not putting path/header/query to model?